core/components/atoms/dropdown/utility.tsx

```diff
@@ -5,9 +5,20 @@ export const getSearchedOptions = (options: any, searchTerm: string) => {
   return result;
 };
 
-export const _isEqual = (arr1: Option[], arr2: Option[]) =>
-  arr1.length === arr2.length &&
-  arr1.every((option, index) => option.value === arr2[index].value || option.label === arr2[index].label);
+const sortList = (arr: Option[]) => {
+  return arr.sort((a, b) => (a.value > b.value ? 1 : b.value > a.value ? -1 : 0));
+};
+
+export const _isEqual = (arr1: Option[], arr2: Option[]) => {
+  const sortedArr1 = sortList(arr1);
+  const sortedArr2 = sortList(arr2);
+  return (
+    sortedArr1.length === sortedArr2.length &&
+    sortedArr1.every(
+      (option, index) => option.value === sortedArr2[index].value || option.label === sortedArr2[index].label
```

Comment:
Please change the variable name from `sortedArr1` to something like `firstSortedList`
and same for `sortedArr2`.